src/util/api/likernft/index.ts

```
@@ -32,10 +32,10 @@ export async function getISCNPrefixByClassId(classId) {
   return decodeURIComponent(doc.id);
 }
 
-
 export async function getNFTClassByClassId(classId) {
   const c = await getNFTQueryClient();
   const client = await c.getQueryClient();
   const res = await client.nft.class(classId);
-  return parseNFTClassDataFields(res);
+  if (!res.class) return new Error('CLASS_NOT_FOUND');
```

should throw instead of return?